src/components/Book.js

```diff
@@ -10,14 +10,22 @@ const Book = () => {
   }, [dispatch]);
 
   const books = useSelector((state) => state.books);
+  console.log(books);
```

Comment:
- [ ] It is often not a good practice to push console logs of this nature as they can expose confidential information in more sophisticated apps